sonar-update-center-common/src/test/java/org/sonar/updatecenter/common/ArtifactTest.java

```diff
@@ -43,9 +43,9 @@ public void compare() {
   @Test
   public void getReleases_orders_releases_by_version() {
     FakeArtifact artifact = new FakeArtifact("fake");
-    artifact.addRelease(Version.create("2.0"));
-    artifact.addRelease(Version.create("1.1"));
-    artifact.addRelease(Version.create("1.5"));
+    artifact.addRelease(new Release(artifact, Version.create("2.0")));
```

Comment:
The pre-existing `addRelease(Version version)` method was making these cases much less verbose.
Should we add it back?

Reply:
I am not a big fan of adding methods to production code just to be used by tests. I added it to Fake artifact